tests/providers/gcp/internal/test_storagetransfer.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the gcp module - storagetransfer.py"""
+
+import typing
+import unittest
+import mock
+
+from tests.providers.gcp import gcp_mocks
+
+
+class GoogleCloudStorageTransferTest(unittest.TestCase):
+  """Test Google Cloud Storage Transfer class."""
+  # pylint: disable=line-too-long
+
+  @typing.no_type_check
+  @mock.patch('boto3.session.Session.get_credentials')
+  @mock.patch('boto3.session.Session._setup_loader')
+  @mock.patch('libcloudforensics.providers.gcp.internal.storagetransfer.GoogleCloudStorageTransfer.GcstApi')
+  def testS3ToGCS(self, mock_gcst_api, mock_loader, mock_creds):
```

Comment:
Would it be worth adding a test that a fail case is handled gracefully? Invalid AWS creds, source object not found, could be potentials.

Reply:
Done